packages/plugins/typescript/graphql-request/tests/__snapshots__/graphql-request.spec.ts.snap

```diff
@@ -238,24 +238,24 @@ export const Feed4Document = \`
 }
     \`;
 
-export type SdkFunctionWrapper = <T>(action: (requestHeaders?:Record<string, string>) => Promise<T>, operationName: string, operationType?: string) => Promise<T>;
+export type SdkFunctionWrapper = <T>(action: (requestHeaders?:Record<string, string>) => Promise<T>, operationName: string, operationType?: string, variables?: any) => Promise<T>;
 
 
-const defaultWrapper: SdkFunctionWrapper = (action, _operationName, _operationType) => action();
+const defaultWrapper: SdkFunctionWrapper = (action, _operationName, _operationType, variables) => action();
```

Comment:
TypeScript is yelling because of that line:

> error TS6133: 'variables' is declared but its value is never read.

Reply:
Seeing the same here after pulling in latest of graphql-request

Reply:
Same, this was also reported [here](https://github.com/dotansimha/graphql-code-generator-community/issues/558) and fixed [here](https://github.com/dotansimha/graphql-code-generator-community/pull/559) but the issue & PR have not gotten any response. It's a very trivial fix and your attention would be appreciated @saihaj or @axe-me